Wyam.Core/Modules/CopyFiles.cs

```diff
@@ -106,7 +106,7 @@ public IEnumerable<IDocument> Execute(IReadOnlyList<IDocument> inputs, IExecutio
                             .Select(file =>
                             {
                                 string destination = _destinationPath == null
-                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(Path.GetDirectoryName(path), Path.GetDirectoryName(file)), Path.GetFileName(file)) 
+                                    ? Path.Combine(context.OutputFolder, PathHelper.GetRelativePath(context.InputFolder, Path.GetDirectoryName(file)), Path.GetFileName(file)) 
```

Comment:
Actually, on second thought, maybe we should support that if we can. I can see a use case where you bring in a folder alongside the Input and Output folders at the root of the project via git submodule or something and then want to copy over stuff inside it.

Would doing a `StartsWith()` to compare the input path with the copy source work for the test?